src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag

```diff
@@ -47,7 +47,7 @@
                     There are no questions for you to answer here!
             </c:when>
             <c:otherwise>
-                <input type="checkbox" name="sendsubmissionemail" checked>
+                <input type="checkbox" name="sendsubmissionemail" checked value="true">
```

Comment:
Indeed it works, but it is like retrofitting the `input` tag to fit our own abstraction `getRequestParamAsBoolean`, when its natural behavior of "on"/"null" (not "off", I just checked again) suffices.